mlflow/store/db_migrations/versions/cfd24bdc0731_update_run_status_constraint_with_killed.py

```diff
@@ -1,7 +1,7 @@
 """Update run status constraint with killed
 
 Revision ID: cfd24bdc0731
-Revises: 89d4b8295536
+Revises: 2b4d017a5e9b
```

Comment:
Do we actually need this docstring?

Reply:
This is auto-generated by alembic, so I guess it was not added with a certain reason. It shouldn't mismatch unless manually edited.

Reply:
Is this the only script that has this issue?

Reply:
can we update `mlflow/store/db_migrations/script.py.mako` and remove `Revision ID` and `Revises`?

Reply:
Ok, that works too

Reply:
@harupy removed. Can you take a look?